### DISCUSSION TITLE
Fullscreen for shader editor

### DISCUSSION THREAD

**Discussion by shahriarlabib000** [godotengine/godot-proposals#10496] [Editor]
So the shader editor coding space is to small. specially on smaller screens. If there was a option to make the shader editor Fullscreen that would be really useful I think.
Or even this red part was collapsible like in the gdscript editor that would also help alot
![Untitled8_20240819214150](https://github.com/user-attachments/assets/0c263ae9-566d-44bf-a68d-3fab8ec852c5)

**Comment by AThousandShips** [comment]
Thank you for your proposal, consolidating in:
* https://github.com/godotengine/godot-proposals/issues/5017

Please add your support there instead (by up-voting) and reacting)
